client/interfaces.go

```diff
@@ -87,6 +87,7 @@ type InstanceServer interface {
 	// Container functions
 	GetContainerNames() (names []string, err error)
 	GetContainers() (containers []api.Container, err error)
+	UpdateInstances(state api.InstancesPut, ETag string) (Operation, error) 
```

Comment:
return should be written as `(op Operation, err error)` for consistency with the rest.